src/collapse/test/collapse.spec.js

```diff
@@ -73,6 +77,9 @@ describe('collapse directive', function() {
     scope.$digest();
     $animate.flush();
     expect(element.height()).toBe(0);
+    expect(element).toHaveClass(collapseConfig.collapseClass);
```

Comment:
I think this is not a useful enough set of expectations in the tests - we need to have a test case where the configuration value is unchanged, and a test case where the configuration value is changed. This needs to be done in two separate tests.